gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/writer/FileAwareInputStreamDataWriter.java

```diff
@@ -161,6 +161,9 @@ public FileAwareInputStreamDataWriter(State state, FileSystem fileSystem, int nu
     this.recoveryHelper = new RecoveryHelper(this.fs, state);
     this.actualProcessedCopyableFile = Optional.absent();
 
+    if (getMetricContext().getInnerMetricContext().getContextAwareMetrics().containsKey(GOBBLIN_COPY_BYTES_COPIED_METER)) {
```

Comment:
Also you can directly getMetricContext().getMetrics() to get the metrics map instead of accessing innerMetricContext?